packages/hash/playwright/tests/page-creation.spec.ts

```diff
@@ -119,10 +119,10 @@ test("user can create page", async ({ page }) => {
    *
    * @see https://app.asana.com/0/1201095311341924/1202033760322934/f
    */
-  await blockContextMenu.locator("text=Load Block").click();
+  await blockContextMenu.locator("text=Re-load block").click();
 
   await expect(
-    blockContextMenu.locator('[placeholder="Load Block from URL..."]'),
+    blockContextMenu.locator('[placeholder="Re-load block"]'),
```

Comment:
This should be "Load Block from URL..." since it's the input's placeholder and not the button 


```suggestion
    blockContextMenu.locator('[placeholder="Load Block from URL..."]'),
```

Reply:
Block should not be capitalized where it appears in the middle of a sentence, outside of the brand name/proper noun "Block Protocol".